### ISSUE TITLE
The MySQL version check cannot determine the current version

### ISSUE THREAD

**Issue by xchs** [contao/installation-bundle#77]
It appears that the [check for the MySQL version](https://github.com/contao/installation-bundle/commit/0514c650bce809bc8081badcdb1cda291f31eecf) may not always correctly determine the current version:

![screenshot-2017-12-15 contao open source cms](https://user-images.githubusercontent.com/1218766/34021409-0399d2d6-e13a-11e7-9e83-6d7726d9a636.png)

Tested with Contao 4.5.0-RC1 and `mysql  Ver 14.14 Distrib 5.1.73, for redhat-linux-gnu (x86_64)`

**Comment by leofeyer** [comment]
You are using MySQL 5.1.73, so the check is correct.

**Comment by fritzmg** [comment]
@leofeyer I think the point was that the version (5.1.73 in this case) is not being shown:
```
aber Sie haben Version .
```

**Comment by leofeyer** [comment]
Thanks, I didn't notice that.

**Comment by leofeyer** [comment]
@xchs What does `SELECT @@version as Version` return on your system?

**Comment by leofeyer** [comment]
And what does `SELECT VERSION()` return?

**Comment by xchs** [comment]
> What does SELECT @@version as Version return on your system?
```
mysql> SELECT @@version;
+------------+
| @@version  |
+------------+
| 5.1.73-log |
+------------+
1 row in set (0.00 sec)
```

> And what does SELECT VERSION() return?
```
mysql> SELECT VERSION();
+------------+
| VERSION()  |
+------------+
| 5.1.73-log |
+------------+
1 row in set (0.01 sec)
```

**Comment by leofeyer** [comment]
Hm, it should work then:

https://github.com/contao/installation-bundle/blob/77da9a750a295b41c775a6ace136d578b1a206a0/src/InstallTool.php#L217-L222

**Comment by xchs** [comment]
I see. Hmm, I'm gonna try it again.

**Comment by xchs** [comment]
So, I've tried again: If I change the Twig template `src/Resources/views/configuration_error.html.twig` and replace there the line
```twig
<p>{{ 'unsupported_version_explain'|trans|format(type, version) }}</p>
```
by
```twig
<p>{{ 'unsupported_version_explain'|trans|format(version) }}</p>
```
I get an output (i.e. the MySQL version is now displayed):

![screenshot-2017-12-15 contao open source cms 1](https://user-images.githubusercontent.com/1218766/34051539-4714f3d0-e1bf-11e7-9d22-998bd6475712.png)

**Comment by leofeyer** [comment]
Seems I forgot to remove the `type` parameters. Thank you for debugging this. 👍

**Comment by leofeyer** [comment]
Fixed in 9297a578c4365e2788956b52d32717f774974e80.
